src/ui/reducers/analyses.ts

```diff
@@ -0,0 +1,151 @@
+import {
+  analysisError,
+  analysisResult,
+  Location,
+  PointDescription,
+  AnalysisEntry,
+} from "@replayio/protocol";
+import { createEntityAdapter, createSlice, PayloadAction } from "@reduxjs/toolkit";
+
+import { AnalysisError } from "protocol/thread/analysis";
+
+enum AnalysisStatus {
```

Comment:
I like where this is headed, but wonder if Analysis should be a first class reducer or if the state should belong to a LogPoint and other application concepts.

This is what Josh and I drew up earlier as a reference
https://gist.github.com/jasonLaster/78fa5b41b7a9c5e9f7986182a60735ed